### PR TITLE
pull about us content from firebase

### PR DESCRIPTION
**Pull request by sandra0503**

### CODE REVIEW

File: src/layouts/BaseLayout.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, {FC} from 'react';
+import React, {FC, useEffect} from 'react';
```
Comment (olierxleben) [inline src/layouts/BaseLayout.tsx:1]: do we need useEffect or is it unused?
```suggestion
import React, {FC} from 'react';
```

File: src/layouts/BaseLayout.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ import {
 } from '@ionic/react';
 
 import {menu} from 'ionicons/icons'
-import {BrowserRouter as Router, Redirect, Route, Switch} from 'react-router-dom';
+import {BrowserRouter as Router, Redirect, Route, Switch, useLocation} from 'react-router-dom';
```
Comment (olierxleben) [inline src/layouts/BaseLayout.tsx:16]:
```suggestion
import {BrowserRouter as Router, Redirect, Route, Switch} from 'react-router-dom';
```

File: src/pages/AboutUsPage.tsx [hunk 3]
```diff
@@ -1,83 +1,38 @@
...
-import logo from '../assets/squirrel-donate.jpg';
+import React, { FC, useContext, useEffect, useState } from 'react';
 import {AppContext} from "../contexts";
 import {valFor} from "../api/language";
 import {IonHeader, IonTitle, IonToolbar} from "@ionic/react";
-
-
-
-
+import { getAboutUsContent, getPrivacyPolicy } from '../api/firebase';
```
Comment (olierxleben) [inline src/pages/AboutUsPage.tsx:5]:
```suggestion
import { getAboutUsContent } from '../api/firebase';
```

unused import

**Review by olierxleben — APPROVED** [review]
Thanks again for the PR. :) Just some additional imports we may not need.

File: src/layouts/BaseLayout.tsx (hunk 1 above)
Comment (sandra0503) [reply src/layouts/BaseLayout.tsx]: Oh no, I just tried something out and forgot to dismiss

Comment (olierxleben) [reply src/layouts/BaseLayout.tsx]: :)

**Pull request merged**